website/docs/cloud-docs/users-teams-organizations/users.mdx

```diff
@@ -31,14 +32,18 @@ After you link your HCP and Terraform Cloud accounts, you can only log in with y
 
 You cannot unlink an autogenerated Terraform Cloud account from your HCP account. You can unlink a pre-existing Terraform Cloud account on the [HCP Account Linking page](#hcp-account-linking) in your **User settings**.
 
+<!-- END: TFC:only -->
+
 ## Creating an Account
 
 To use Terraform Cloud or Enterprise, you must create an account through one of the following methods:
 
 - **Invitation Email:** When a user sends you an invitation to join an existing Terraform Cloud organization, the email includes a sign-up link. After you create an account, you will automatically join that organization and can begin using Terraform Cloud.
 - **Sign-Up Page:** Creating an account requires a username, an email address, and a password. For Terraform Cloud, go to [`https://app.terraform.io/public/signup/account`](https://app.terraform.io/public/signup/account). For Terraform Enterprise, go to `https://<TFE HOSTNAME>/public/signup/account`. After you create an account, you will not belong to any organizations. To begin using Terraform Cloud, you can either [create an organization](/cloud-docs/users-teams-organizations/organizations#creating-organizations) or ask an organization owner to send you an invitation email to join their organization.
```

Comment:
```suggestion
- **Sign-Up Page:** Creating an account requires a username, an email address, and a password. For Terraform Cloud, go to [`https://app.terraform.io/public/signup/account`](https://app.terraform.io/public/signup/account). For Terraform Enterprise, go to `https://<TFE HOSTNAME>/public/signup/account`. After you create an account, you do not belong to any organizations. To begin using Terraform Cloud, you can either [create an organization](/cloud-docs/users-teams-organizations/organizations#creating-organizations) or ask an organization owner to send you an invitation email to join their organization.
```

Suggest changing "will" to "do" to avoid future tense.